packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -47,7 +49,7 @@ export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
     return (
-        <AppHeaderDropdown buttonChildren={isMobile ? <Account /> : user.name} startIcon={isMobile ? undefined : <Account />}>
+        <AppHeaderDropdown buttonChildren={buttonChildren ?? (isMobile ? <Account /> : user.name)} startIcon={isMobile ? undefined : <Account />}>
```

Comment:
I'm not sure how this should behave on mobile: Should buttonChildren always replace it, or only on Desktop (= user.name)

Reply:
I think in both cases